app/models/test_passage.rb

```diff
@@ -0,0 +1,59 @@
+class TestPassage < ApplicationRecord
+  WIN_PERCENT = 85
+  belongs_to :user
+  belongs_to :test
+  belongs_to :current_question, class_name: 'Question', optional: true
+
+  before_validation :set_current_question
+
+  def accept!(answer_ids)
+    self.correct_questions += 1 if correct_answer?(answer_ids)
+    save!
+  end
+
+  def completed?
+    current_question.nil?
+  end
+
+  def percent_result
+    (correct_questions * 100 / test.questions.count).to_i
+  end
+
+  def result_true?
+    true if percent_result >= WIN_PERCENT
```

Comment:
```suggestion
  percent_result >= WIN_PERCENT
```
сравнение само по себе возвращает булево